tldw_Server_API/tests/External_Sources/test_connectors_worker_reference_sync.py

```
@@ -0,0 +1,184 @@
+
+    class _FakeMDB:
+        def __init__(self):
+            self.closed = False
+
+        def close_connection(self):
+            self.closed = True
+
+    created_media_db = _FakeMDB()
+
+    monkeypatch.setattr(dbmod, "get_db_pool", _fake_get_db_pool)
+    monkeypatch.setattr(orgs, "list_memberships_for_user", lambda user_id: [])
```

Ruff (0.15.7)

[warning] Unused lambda argument: `user_id`

(ARG005)